src/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { createStore } from 'redux';
+import { devToolsEnhancer } from 'redux-devtools-extension';
+import reducer from './books';
+
```

Comment:
You've done a good job. I noticed that you have chosen to implement some the requirements using some other approaches. Please note that the reason why a specific approach to implementing some of these features is required is in order to make sure some of the core concepts in the curriculum are understood. That been said, please see the required changes below:

- [ ] As mentioned by the previous reviewer, please make use of Combine reducers as required. You may not see the need for it at the moment because you only have one reducer. You'll definitely see the need in the coming milestones as you'll be required to implement another reducer.